src/server/modes/base/maintenance/players/update.ts

```diff
@@ -643,6 +643,7 @@ export default class GamePlayersUpdate extends System {
             player.planestate.stealthed = false;
             player.times.lastStealth = now;
             player.delayed.BROADCAST_EVENT_STEALTH = true;
+            this.delay(BROADCAST_PLAYER_UPDATE, player.id.current);
```

Comment:
It should be replaced with `player.delayed.BROADCAST_PLAYER_UPDATE = true;` to prevent double identical PLAYER_UPDATE broadcast (channels don't check event for unique).

Reply:
Oh wait, "before PLAYER_FIRE" won't work so. I need to think about it.